otx/algorithms/classification/adapters/mmcls/optimizer/lars.py

```diff
@@ -1,6 +1,8 @@
+"""Module for defining LARS optimizer for classification task."""
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
+# pylint: disable=too-many-arguments, too-many-locals, too-many-branches
```

Comment:
Please disabling pylint as minimum as possible. How about setting in on necessary functions or classes?